person.rb

```diff
@@ -35,9 +35,9 @@ def of_age?
   end
 end
 
-person = Person.new(22, 'maximilianus')
-puts person.correct_name
-capitalized_person = CapitalizaDecorator.new(person)
-puts capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts capitalized_trimmed_person.correct_name
+# person = Person.new(22, 'maximilianus')
+# puts person.correct_name
+# capitalized_person = CapitalizaDecorator.new(person)
+# puts capitalized_person.correct_name
+# capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
+# puts capitalized_trimmed_person.correct_name
```

Comment:
- [ ] Let's clean up our code a bit, it's not advisable to leave commented code because the codebase looks dirty.